package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "seek-style-guide",
-  "version": "15.1.0",
+  "version": "0.0.0-development",
```

Comment:
In semantic-release land, you don't check the version number into source control, it becomes an artifact of the build. This is the standard placeholder used instead.